ground/gcs/src/plugins/config/configgadgetwidget.cpp

```diff
@@ -58,8 +58,11 @@ ConfigGadgetWidget::ConfigGadgetWidget(QWidget *parent) : QWidget(parent)
 {
     setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
 
+    this->setObjectName(QString("ConfigWidget"));
```

Comment:
This can probably be dropped. I still need to verify for sure, but referring by class name seems to work, too. Names are probably not needed unless there's multiple instances, which in this case shouldn't.